### PR TITLE
[koa-better-body] Add types

### PR DESCRIPTION
**Pull request by Dandan1314**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
<[formidable instance](https://github.com/tunnckoCoreLabs/koa-better-body/blob/master/recipes/formidable/README.md),    [multipart](https://github.com/tunnckoCoreLabs/koa-better-body/blob/master/recipes/multipart/README.md),        [form](https://github.com/tunnckoCoreLabs/koa-better-body/blob/master/recipes/form/index.js)>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/koa-better-body/index.d.ts [hunk 1]
```diff
@@ -25,6 +26,11 @@ declare namespace KoaBetterBody {
     decodeURIComponent?: any;
     maxKeys?: number;
     buffer?: boolean;
+    QueryString?: string;
```
Comment (danwbyrne) [inline types/koa-better-body/index.d.ts]: I believe this should just be `querystring`.

File: types/koa-better-body/index.d.ts [hunk 2]
```diff
@@ -25,6 +26,11 @@ declare namespace KoaBetterBody {
...
+    encoding?: string;
```
Comment (danwbyrne) [inline types/koa-better-body/index.d.ts]: I'm not seeing `encoding` as an option anywhere in the [defaultOptions](https://github.com/tunnckoCoreLabs/koa-better-body/blob/e5f6f677a93b6acb39d1173dea3d5cf2f917c035/utils.js#L20). Perhaps you are trying to write definitions for *[this](https://github.com/guileen/koa-better-body)* `koa-better-body`. :thinking:

**Review by danwbyrne — CHANGES_REQUESTED** [review]
While some of your types line up with the options in https://github.com/tunnckoCoreLabs/koa-better-body/blob/master/utils.js#L20 I am also not seeing some of them? Can you find a reference?

File: types/koa-better-body/index.d.ts (hunk 2 above)
Comment (Dandan1314) [reply types/koa-better-body/index.d.ts]: Sorry my poor English. <[Here](https://github.com/tunnckoCoreLabs/koa-better-body/blob/master/recipes/multipart/README.md)> is a reference of the options.🤔

`encoding`, `uploadDir `,  `keepExtensions` are using as options in _multipart_

File: types/koa-better-body/index.d.ts (hunk 1 above)
Comment (Dandan1314) [reply types/koa-better-body/index.d.ts]: Well, it's _querystring_. That's my mistake.

Comment (Dandan1314) [reply types/koa-better-body/index.d.ts]: Well, it's `querystring`. That's my mistake.

**Pull request closed without merging**